src/index.ts

```diff
@@ -388,6 +418,31 @@ export function graphqlHTTP(options: Options): Middleware {
       errors: result.errors?.map(formatErrorFn),
     };
 
+    if (isAsyncIterable(executeResult)) {
+      response.setHeader('Content-Type', 'multipart/mixed; boundary="-"');
+      sendPartialResponse(pretty, response, formattedResult);
+      for await (let payload of executeResult) {
```

Comment:
@robrichard What happens when the request is terminated by the client (i.e. I navigate away from the page or close the browser)? Would it make sense to add an event listener to response for the `close` event and call the AsyncIterator's `return` method inside?

Reply:
Yes, I think that makes sense. I will implement that.

Reply:
I added a handler for the close event to call `return` on the underlying async iterable. 

@IvanGoncharov this required disabling some lint rules (see https://github.com/graphql/express-graphql/pull/583/commits/d883ae71eeacb9b148f32d01f68b20c831ceca07#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R419). Do you think this is ok or do you have a better way to implement this?